### ISSUE TITLE
Error when using assistant

### ISSUE THREAD

**Issue by madchicken** [64bit/async-openai#234]
I am getting an error when trying to start a thread for an assistant:

```
Error: ApiError(ApiError { message: "Invalid type for 'parallel_tool_calls': expected a boolean, but got null instead.", type: Some("invalid_request_error"), param: Some("parallel_tool_calls"), code: Some("invalid_type") })
```

I think OpenAI changed something VERY recently, since yesterday this was not happening.
The example under `assistants` is failing with the same error

**Comment by 64bit** [comment]
Oh no, thats no good. 

Thank you for reporting the issue.

From the error message you have provided, it seems that skipping serialization if parallel_tool_calls is None would prevent sending null to their API and fix it?

**Comment by madchicken** [comment]
Please, disregard my previous message, the error is way below, when running the thread request:
```
        //create a run for the thread
        let run_request = CreateRunRequestArgs::default()
            .assistant_id(assistant_id)
            .build()?;
```
and I was able to fix it by doing what you said:
```
        //create a run for the thread
        let run_request = CreateRunRequestArgs::default()
            .assistant_id(assistant_id)
            .parallel_tool_calls(false)
            .build()?;
```

**Comment by 64bit** [comment]
Thank you for the issue and PR, the fix is released in v0.23.3
